rest/config.go

```diff
@@ -1000,9 +1003,11 @@ func (sc *ServerContext) fetchAndLoadDatabase(dbName string) (found bool, err er
 		// TODO: This code is mostly copied from fetchConfigs, move into shared function with DbConfig REST API work?
 
 		// inherit properties the bootstrap config
-		cnf.Server = &sc.config.Bootstrap.Server
+		// cnf.Server = &sc.config.Bootstrap.Server
 		cnf.CACertPath = sc.config.Bootstrap.CACertPath
-		cnf.Bucket = &bucket
+
+		bucketCopy := bucket
+		cnf.Bucket = &bucketCopy
```

Comment:
I didn't think that empty string was a valid bucket name - is the pointer handling in cnf.Bucket just to avoid custom handling for mergo?

Reply:
No but most of the `DatabaseConfig` fields are inherited from `DbConfig` due to complexities in ripping out or changing DbConfig due to retaining 2.x handling.